core/sources/domains.move

```diff
@@ -660,7 +701,7 @@ module aptos_names::domains {
         };
 
         // Only the owner or the registered address can clear the address
-        let (is_owner, token_id) = is_owner_of_name(signer_addr, subdomain_name, domain_name);
+        let (is_owner, token_id) = is_token_owner(signer_addr, subdomain_name, domain_name);
```

Comment:
we don't check expiration here cause even expire, owner should still be able to clear it